project/Build.scala

```diff
@@ -12,9 +12,9 @@ import com.typesafe.sbt.sdlc.Plugin._
 
 object SlickBuild extends Build {
 
-  val slickVersion = "3.2.0-SNAPSHOT"
+  val slickVersion = "3.2.0-M2"
```

Comment:
I'd prefer to keep SNAPSHOT here so you don't accidentally publish a non-snapshot version.